src/ui/LiveSplit.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+import localforage from "localforage";
```

Comment:
I'm not sure if we want localforage or not. If we manually talk to IndexedDB we could have tables for splits and for layouts, which would help us for the run and layout selection view.